{"uuid": "56cbda0f-8b7e-44e2-b345-46a749d2c444", "children": ["42866991-b730-49d0-bc20-f5023a504200"], "befores": [{"name": "page", "status": "passed", "start": 1745389318870, "stop": 1745389320578}], "afters": [{"name": "page::0", "status": "broken", "statusDetails": {"message": "Exception: Page.close: Connection closed while reading from the driver\n", "trace": "  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\Python\\uiautotest\\tests\\conftest.py\", line 19, in page\n    page.close()\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 9842, in close\n    self._sync(\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\playwright\\_impl\\_page.py\", line 814, in close\n    raise e\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\playwright\\_impl\\_page.py\", line 809, in close\n    await self._channel.send(\"close\", locals_to_params(locals()))\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 61, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Python\\uiautotest\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 528, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "start": 1745389331046, "stop": 1745389331056}], "start": 1745389318870, "stop": 1745389331056}